backend/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "ES2016",
+    "module": "CommonJS",
+    "outDir": "./dist",
```

Comment:
commonJs or module? figure this out, we don't want to go backwards we want to use "module" in the nodejs app but try to figure out what option to give in the tsconfig
https://stackoverflow.com/questions/55471795/what-is-module-option-in-tsconfig-used-for